ts/Dashboards/Layout/Row.ts

```diff
@@ -342,6 +342,27 @@ class Row extends GUIElement {
         };
     }
 
+    /**
+     * Get the row's options.
+     * @returns
+     * The JSON of row's options.
+     *
+     * @internal
+     *
+     */
+    public getOptions(): DeepPartial<Row.Options> {
+        const row = this,
+            cells = [];
+
+        for (let i = 0, iEnd = row.cells.length; i < iEnd; ++i) {
+            cells.push(row.cells[i].getOptions());
+        }
+
+        return {
+            cells
+        };
```

Comment:
Maybe let's serialise some other options of row, like `id` or `style` (it is still in the API)?